### DISCUSSION TITLE
Creating New Marker in Week 2 In-Class Lab Assignment

### DISCUSSION THREAD

**Discussion by alizezila** [albertkun/21S-ASIAAM-191A#53] [Help Desk] [answered]
Hey all! Sorry if this is a silly question, I just have been fiddling with it for some time and am not sure what's wrong? I am trying to place a new marker on the leaflet map through JS and am copying the code from the lab assignment 

//JavaScript let variable declaration to create a marker
let marker = L.marker([34.0709, -118.444]).addTo(map)
		.bindPopup('Math Sciences 4328 aka the Technology Sandbox<br> is the lab where I work in ')
		.openPopup();

and changing the coordinates to a new point. This is what my code looks like for example 

let newmarker = L.newmarker([34.00361978702228, -118.04614394751376]).addTo(map) 
.bindPopup('new marker') 
.openPopup();

So I'm just copying and pasting, but not getting a new marker. I have pretty much no coding experience so if anyone can help I would really appreciate it! Thanks!

**Comment by albertkun** [comment, marked as answer]
@alizezila! So close!!!!

You actually should not rename `L.marker` to `L.newmarker` because `L.marker` is a leaflet class [https://leafletjs.com/reference-1.7.1.html#marker](https://leafletjs.com/reference-1.7.1.html#marker)

Your correct complete marker function should look like this:
```js
let newmarker = L.marker([34.00361978702228, -118.04614394751376]).addTo(map)
.bindPopup('new marker')
.openPopup();
```
Also, just FYI you can paste your code with formatting by clicking the "insert code" button.  You can also use apostrophes. For more info: [https://guides.github.com/features/mastering-markdown/#](https://guides.github.com/features/mastering-markdown/#)

p.p.s. remember to mark this question as answered if it helps! 😄

**Reply by alizezila** [reply]
Thanks @albertkun ! Looks great now 🤩

**Reply by albertkun** [reply]
You are welcome! Glad it worked out @alizezila ! Can't wait to see it! 😃
